data/remnant missions.txt

```diff
@@ -238,14 +316,39 @@ event "remnant: surveillance end"
 	system "Parca"
 		add fleet "Korath Raid" 10000
 
-
+		
+mission "Remnant: Clean-up Bounty"
+	job
+	repeat
+	name "Remnant clean-up bounty"
+	description "Clean up a fleet of Korath chasers harassing ships in Remnant territory, then return to <planet>. Remember to salvage whatever you can."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Defense 2: done"
+		random < 30
+	npc kill
+		government "Korath (Berserk)"
+		personality coward target uninterested marked staying
+		system
+			distance 1 2
+		fleet 4
+			names "korath"
+			cargo 3
+			variant
+				"Korath Chaser"
+		dialog "You have destroyed the Korath ships. You can now return to <planet>."
+	on complete
+		payment 100000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ships, and gives you the agreed-upon payment of <payment>."
 
 mission "Remnant: Key Stones"
 	name "Keystones"
 	description "The manager of the outfitter on <planet>, a Remnant world, has offered to pay you six million credits in exchange for a shipment of fifty Quantum Keystones (which you will have to purchase from the Hai)."
 	source "Viminal"
 	to offer
 		has "First Contact: Remnant: offered"
+		has "remnant blood test pure"
```

Comment:
I would disagree, since some missions may provide clearance to land.

Reply:
Just in case the player somehow manages to land again in the future, perhaps via missions that provide clearance. 

Given that this is a rather large batch of missions and stuff, chances are people will get more interested in it, and maybe even make more missions for it. Easier to setup the missions properly now than hope that someone adding a mission that provides a clearance down the road thinks to go in and set all of these)

Reply:
I don't believe that Remnant planets decline bribes either, which should be the case.

Reply:
The Remnant as a whole and their planets in particular have been given `Bribe 0` which should stop that.